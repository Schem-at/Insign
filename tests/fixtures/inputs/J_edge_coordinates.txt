@negative=rc([-100,-50,-25],[100,50,25])
#negative:type="spans_origin"

@large=rc([1000000,2000000,3000000],[1000063,2000031,3000015])  
#large:coordinates="very_large_values"

@zero_size=rc([10,10,10],[10,10,10])
#zero_size:warning="single_point_region"

@backwards=rc([20,20,20],[10,10,10])
#backwards:note="coordinates_will_be_normalized"
