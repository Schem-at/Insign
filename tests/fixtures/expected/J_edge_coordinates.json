{
  "backwards": {
    "bounding_boxes": [
      [
        [
          10,
          10,
          10
        ],
        [
          20,
          20,
          20
        ]
      ]
    ],
    "metadata": {
      "note": "coordinates_will_be_normalized"
    }
  },
  "large": {
    "bounding_boxes": [
      [
        [
          1000000,
          2000000,
          3000000
        ],
        [
          1000063,
          2000031,
          3000015
        ]
      ]
    ],
    "metadata": {
      "coordinates": "very_large_values"
    }
  },
  "negative": {
    "bounding_boxes": [
      [
        [
          -100,
          -50,
          -25
        ],
        [
          100,
          50,
          25
        ]
      ]
    ],
    "metadata": {
      "type": "spans_origin"
    }
  },
  "zero_size": {
    "bounding_boxes": [
      [
        [
          10,
          10,
          10
        ],
        [
          10,
          10,
          10
        ]
      ]
    ],
    "metadata": {
      "warning": "single_point_region"
    }
  }
}